samples/contacturi/src/main/java/com/bumptech/glide/samples/contacturi/MainActivity.java

```diff
@@ -48,6 +67,18 @@ public void onClick(View v) {
         startActivityForResult(intent, REQUEST_CONTACT);
       }
     });
+
+    findViewById(R.id.button_find).setOnClickListener(new View.OnClickListener() {
+      @Override
+      public void onClick(View v) {
+        Uri uri = Uri.withAppendedPath(ContactsContract.PhoneLookup.CONTENT_FILTER_URI,
+                                       Uri.encode(numberEntry.getText().toString()));
+        GlideRequests glideRequests = GlideApp.with(MainActivity.this);
+        RequestOptions originalSize = new RequestOptions().override(Target.SIZE_ORIGINAL);
+
+        glideRequests.load(uri).apply(originalSize).into(imageViewLookup);
```

Comment:
You can inline this:

GlideApp.with(MainActivity.this)
  .load(uri)
  .override(Target.SIZE_ORIGINAL)
  .into(imageView);